tests/passes/cell-share/bounded.futil

```diff
@@ -1,4 +1,4 @@
-// -p cell-share -x cell-share:bounds=2,3,4 -p remove-ids
```

Comment:
What does it mean to have a negative bound?

Reply:
it means always share that cell type.

Reply:
Got it. If it's not already documented, we should say that in the comment for the pass.